<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="cluster_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Cluster
(raw '.h5')</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Save">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Load">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Load
(clustered '.h5')</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="save_progress">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>228</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>         Files         </string>
    </property>
   </widget>
   <widget class="QProgressBar" name="load_progress">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>268</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>350</y>
      <width>469</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item row="3" column="3">
      <widget class="QSpinBox" name="ToF_min">
       <property name="maximum">
        <number>16667</number>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QCheckBox" name="wADC_filter">
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_18">
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string> Grids</string>
       </property>
      </widget>
     </item>
     <item row="5" column="6">
      <widget class="QSpinBox" name="grid_max">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>40</number>
       </property>
       <property name="value">
        <number>40</number>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="wADC_max">
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="value">
        <number>12288</number>
       </property>
      </widget>
     </item>
     <item row="0" column="5" colspan="2">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>Coordinates</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QSpinBox" name="gADC_min">
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="value">
        <number>400</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_36">
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>   Active</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_17">
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string> Wires</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QCheckBox" name="gADC_filter">
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="QSpinBox" name="grid_min">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>80</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="wADC_min">
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="value">
        <number>500</number>
       </property>
      </widget>
     </item>
     <item row="6" column="5">
      <widget class="QCheckBox" name="wire_filter">
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="5">
      <widget class="QSpinBox" name="wire_max">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="value">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_16">
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>      Max</string>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QSpinBox" name="module_max">
       <property name="maximum">
        <number>8</number>
       </property>
       <property name="value">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QCheckBox" name="ToF_filter">
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="6">
      <widget class="QCheckBox" name="grid_filter">
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QSpinBox" name="wire_min">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_15">
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>       Min</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLabel" name="label_21">
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Wires</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Charge [ADC channels]</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QSpinBox" name="gADC_max">
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="value">
        <number>12288</number>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QLabel" name="label_22">
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Grids</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QSpinBox" name="ToF_max">
       <property name="maximum">
        <number>16667</number>
       </property>
       <property name="value">
        <number>16667</number>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_12">
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string> ToF [µs]</string>
       </property>
      </widget>
     </item>
     <item row="6" column="4">
      <widget class="QCheckBox" name="module_filter">
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QSpinBox" name="module_min">
       <property name="maximum">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Module</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>-10</y>
      <width>551</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>Multi-Grid: VMM</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>330</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>          Filter          </string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>481</width>
      <height>61</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>12</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>71</width>
      <height>121</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>12</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>PHS</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>150</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Coincidences</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PHS_1D_button">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>1D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Coincidences_2D">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>170</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>2D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PHS_2D_button">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>210</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>2D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Coincidences_3D">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>210</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>3D</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>150</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Miscellaneous</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>130</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>          Analysis          </string>
    </property>
   </widget>
   <widget class="QRadioButton" name="write">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Write</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="append">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Append</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="cluster_progress">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="timestamp_button">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>170</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Timestamp</string>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>481</width>
      <height>61</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>12</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_43">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>560</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ToF:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_45">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>580</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>PHS:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tofBins">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>560</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="phsBins">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>580</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>50</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_50">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>540</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Bins</string>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>540</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>12</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>540</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>12</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>560</y>
      <width>87</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>MG.24</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>580</y>
      <width>87</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>MG.SEQ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>540</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>          Detector          </string>
    </property>
   </widget>
   <widget class="QPushButton" name="Timestamp_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>550</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>HELP</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="data_sets_browser">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Data sets:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>499</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
